functions-and-scope.js

```diff
@@ -82,3 +147,19 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // highestGrade(grades) geeft 9
 // highestGrade([6, 4, 5]) geeft 6
 // highestGrade([8, 9, 4, 6, 10]) geeft 10
+
+function highestGrade(grades) {
+    let highestGrade = 0;
```

Comment:
Het is beter om als default waarde altijd het eerste cijfer van de array te pakken. Cijfers zullen in dit voorbeeld altijd hoger dan 0 zijn, maar voer je functie eens uit met een array als `[-1, -2, -4, -1]`. Dan zal er `0` uitkomen terwijl die waarde niet eens in de array voorkomt 😉